incubator/sapho/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for sapho.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image: sapho/ops-docker-tomcat:8-jre8
+
+mysql:
+  mysqlUser: root
```

Comment:
This setting causes MySQL to crash, `root` is the default user created so with this it tries to create the root user twice. It should start up fine without this set.